website/content/docs/platform/aws/lambda-extension.mdx

```diff
@@ -243,6 +243,10 @@ request will be forwarded to Vault and the response returned and cached.
 Currently, the cache key is a hash of the request URL path, headers, body, and
 token.
 
+~> **Warning!** Since the Vault Lambda Extension's cache key includes the headers of the request
+to the proxy, any usage of distributed tracing headers in such request, like `traceparent`, may render all requests
+unique and prevent cache hits.
```

Comment:
```suggestion
<Warning title="Distributed tracing headers may negate the cache">

  The Vault Lambda Extension cache key includes headers from proxy requests. As
  a result, distributed tracing headers like `traceparent` can lead to
  individualized hashes that make repeated requests appear unique and cause cache
  misses.

</Warning>
```

Style correction: write in active voice, avoid possessives, use tag-style aside notation